packages/vitest-pool-workers/src/worker/index.ts

```diff
@@ -31,15 +29,25 @@ function structuredSerializableParse(value: string): unknown {
 
 globalThis.Buffer = Buffer; // Required by `vite-node/source-map`
 
+// Mock Service Worker needs this
+globalThis.BroadcastChannel = class {};
+
 globalThis.process = process; // Required by `vite-node`
 process.argv = []; // Required by `@vitest/utils`
 let cwd: string | undefined;
 process.cwd = () => {
 	assert(cwd !== undefined, "Expected cwd to be set");
 	return cwd;
 };
+// Required by vitest/worker
+// @ts-expect-error We don't actually implement `process.memoryUsage()`
+process.memoryUsage = () => ({});
```

Comment:
So it's fine with the stub implementation? Could we upstream a fix to remove the requirement?